### ISSUE TITLE
The app is empty?

### ISSUE THREAD

**Issue by RepressedNerd** [alexzielenski/Mousecape#136]
Hello,

I'm not sure if there's something extra I need to do, but I downloaded the mater zip file and the application zip file. But the app is completely empty.

If there's something I need to do, could you please walk me through the steps?

Thanks in advance!

![image](https://user-images.githubusercontent.com/67737702/93848836-0f758f00-fcee-11ea-96ed-e9a8709f0bf8.png)

**Comment by NordJan** [comment]
Yes, you need to "populate" the application with one (or more) cursorsets, in MouseCape-speak called .cape files.
So, get a cursorset, probaly as .zip and unzip, a .cape should be produced. Import this .cape in MouseCape (Cmd+O). If all goed well, it should appear within the app.
Let me know how you go with this.
Cheers, Nord-Jan Vermeer
![Screenshot 2020-09-22 at 08 48 54](https://user-images.githubusercontent.com/8850154/93851511-8be97d00-fcb0-11ea-8f4d-b48761865be6.jpg)

**Comment by TheHollenVAI** [comment]
sednd me

**Comment by NordJan** [comment]
> sednd me

If you want assistance, it helps if you are more clear about your request.

**Comment by TeslaOwnerTips** [comment]
M1 Man Mini
I have the same. empty window. I imported com.maxrudberg.svanslosbluehazard.cape
Both the app and the cape are in the downloads folder.
Helper installed.

**Comment by NordJan** [comment]
I think it is better to have the application in the Application folder.
How did you import the .cape file and the window remains empty?

**Comment by TeslaOwnerTips** [comment]
Done. Same result. Window empty.
![Untitled 2](https://user-images.githubusercontent.com/78335749/115403985-e69d7580-a1ba-11eb-8af9-165ae8a489bf.png)

**Comment by NordJan** [comment]
To further narrow this issue down:
- what size is your com.maxrudberg.svanslosbluehazard.cape? It should be around 2.3MB. If not, try downloading it again by right-clicking on the link 'View raw' and "Save as ...'.
- did you try to import this .cape with Cmd+O?

**Comment by TeslaOwnerTips** [comment]
That was it :) Small file.
I initially used this one from the code folder.
https://github.com/alexzielenski/Mousecape/blob/master/Mousecape/com.maxrudberg.svanslosbluehazard.cape
I am not using the file linked to from the readme.
